runtime/tools/spec-builder/index.js

```diff
@@ -1,8 +1,28 @@
-const fs = require('fs');
+#!/usr/bin/env node
 const GenesisBuilder = require('./src/genesis-builder');
-const customSpec = require('./customSpec.json');
+const fs = require('fs');
+const curPath = process.cwd();
+
+let argv = require('yargs')
```

Comment:
I don't think that we need a separate `command` called `build-spec`. 
I would like to be able to do `./node_modules/@as-substrate/spec-builder -f spec.json -o raw.json`

There is no need to configure a command. 
You can try and parse the args only.

You can check for example here: https://github.com/trufflesuite/ganache-cli/blob/develop/cli.js
simply by typing `ganache` it will start the server without any command. The only thing that the user can provide are the options similar to us -> the user does not have to specify a command but simply execute the `index.js` file and pass the parameters `-f` and `-o` (and their  aliases)